api/FileReaderSync.json

```
@@ -249,103 +249,59 @@
           }
         }
       },
-      "service_workers_support": {
+      "worker_support": {
         "__compat": {
-          "description": "Service workers support",
+          "description": "Available in workers (except for service workers)",
```

The linter won't accept any description but "Available in workers" here (though I appreciate what you're getting at).

```suggestion
          "description": "Available in workers",
```